TF_Settings_Web/src/Styles/_variables.scss.d.ts

```diff
@@ -6,6 +6,7 @@ export interface GlobalCSS {
     ultraleapGreen: string;
     lightGreyGradient: string;
     darkGreyGradient: string;
+    handNotFoundHeight: string;
```

Comment:
Does this need to be here and in Calibrate.scss.d.ts?

Reply:
I think this might have been leftover from a merge conflict - it's gone now